pkg/code-generator/docgen/templates/markdown/proto_file_template.go

```diff
@@ -44,7 +44,7 @@ func ProtoFileTemplate(project *model.Project, docsOptions *options.DocsOptions)
 
 {{ end}}
 
-##### Source File: {{ githubLinkForFile "master" .Name }}
+##### Source File: {{ githubLinkForFile "main" .Name }}
```

Comment:
does this need to be updated in `pkg/code-generator/docgen/templates/restructured/proto_file_template.go` too?